config/database.yml

```diff
@@ -4,13 +4,13 @@ development:
   database: myflix_development
   host: localhost
   pool: 5
-  username: jdhang
-  password: test 
+  username: jhang
+  password:
 
 test:
   adapter: postgresql
   encoding: unicode
   database: myflix_test
   pool: 5
-  username: jdhang
-  password: test
\ No newline at end of file
+  username: jhang
+  password:
```

Comment:
Your `database.yml` file should not be committed to your repo. You should commit a `database.yml.sample` which contains an outline of what is expected, but never your live yml file.